skopt/tests/test_opt.py

```diff
@@ -0,0 +1,32 @@
+from sklearn.utils.testing import assert_array_equal
+from sklearn.utils.testing import assert_raises
+
+from skopt import dummy_minimize, gp_minimize, forest_minimize, gbrt_minimize
+
+from skopt.benchmarks import branin
+
+
+def test_init_vals():
+    optimizers = [
+        dummy_minimize,
+        gp_minimize,
+        forest_minimize,
+        gbrt_minimize
+    ]
+    x0 = [[1, 2], [3, 4], [5, 6]]
+    y0 = map(branin, x0)
```

Comment:
This will be a map object in Python3, hence you need to explicitly convert it to a list internally.